dracut/repartition/endless-repartition.sh

```diff
@@ -29,8 +29,15 @@
 #
 # Based on code from dracut-modules-olpc.
 
+root_part=$(systemctl show -p What sysroot.mount)
```

Comment:
Yes, it's tested.

sysroot.mount is a core part of the boot process, nothing new, you can see that we already reference it in the service file. It is indeed autogenerated.